pkg/parser/parser.go

```diff
@@ -0,0 +1,33 @@
+package parser
+
+import (
+	"bytes"
+	_ "embed"
+	"log"
+	"text/template"
+)
+
+var (
+	//go:embed cse_cmd.sh.gtpl
+	bootstrapTrigger         string
+	bootstrapTriggerTemplate = template.Must(template.New("triggerBootstrapScript").Parse(bootstrapTrigger)) //nolint:gochecknoglobals
```

Comment:
the only way to use go:embed, as Karpenter does https://github.com/Azure/karpenter/blob/057ac97ff34fe57195e00ea9e8cb70148656b447/pkg/providers/imagefamily/bootstrap/aksbootstrap.go#L225 is to use global variables.

from official Golang docs:

`the variable declared with go:embed must be a package-level variable. The directive allows you to embed files or directories into your Go program during the build process, and this embedding happens at the package level.`

if there is unanimous pushback on suppressing the golint, we can go back to using the more verbose method. can also revisit this when this is closer to production.